### PR TITLE
Feat/time fields step

### PR DESCRIPTION
**Pull request by wysher**
#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [x] Admin
- [ ] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

With this PR It will be pausible to set step for time and datetime fields through [Model]settings.json file or on frontend on field settings in advanced tab.

### CODE REVIEW

**Comment by wysher** [comment]
@soupette thanks. I thought about content-manager as well, but IMO step should not change. In most cases step will be the same for entire app, but there may be some models with another step settings.

Looking forward for feedback.

**Comment by soupette** [comment]
> @soupette thanks. I thought about content-manager as well, but IMO step should not change. In most cases step will be the same for entire app, but there may be some models with another step settings.
> 
> Looking forward for feedback.

Well such specific setting could be exported in a config file ([like this one](https://github.com/strapi/strapi/blob/master/packages/strapi-plugin-content-manager/admin/src/components/CustomTable/DATE_FORMATS.js)) so it is easier for a user to modify until we develop a global field API. If we do so I think adding a `config.js` file at the root of the plugin's `admin/src` folder would be great.

**Comment by alexandrebodin** [comment]
@Aurelsicoko Could you shime in on this plz :)

File: docs/3.0.0-beta.x/concepts/models.md [hunk 1]
```diff
@@ -166,6 +166,7 @@ If you're using SQL databases, you should use the native SQL constraints to appl
 - `index` (boolean) — adds an index on this property, this will create a [single field index](https://docs.mongodb.com/manual/indexes/#single-field) that will run in the background (_only supported by MongoDB_).
 - `max` (integer) — checks if the value is greater than or equal to the given minimum.
 - `min` (integer) — checks if the value is less than or equal to the given maximum.
+- `step` (integer) — for time and datetime content types defines step in minutes (value must be divisable by 60).
```
Comment (lauriejim) [inline docs/3.0.0-beta.x/concepts/models.md:169]: If your PR is merged after this one - https://github.com/strapi/strapi/pull/5674
Can you please add this option in the `Exceptions` section.
Thank you.

**Comment by alexandrebodin** [comment]
Hi, seeing all the conflicts and changes made recently I think we should close this issue @wysher If you still feel there is a need please open a feature request issue to start discussing the implementation and need first :)

**Comment by wysher** [comment]
I'll be happy to open new PR, but there's still no update of preffered approach. 

@alexandrebodin @soupette can I have your help with this? There are Two proposals:

1. set step through `content-type-builder` in model.settings.json file (i preffer this one).
2. set step through `content-manager`.

**Comment by alexandrebodin** [comment]
@wysher Let's start first with an open github discussion so we get enough feedbakc from other users :)

**Pull request closed without merging**